fastlane/lib/assets/completions/completion.bash

```diff
@@ -0,0 +1,16 @@
+_fastlane_complete() {
+  COMPREPLY=()
+  local word="${COMP_WORDS[COMP_CWORD]}"
+  local completions=""
+
+  # look for Fastfile either in this directory or fastlane/ then grab the lane names
+  if [[ -e "Fastfile" ]]; then
+    completions=`cat Fastfile | grep "lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
+  elif [[ -e "fastlane/Fastfile" ]]; then
+    completions=`cat fastlane/Fastfile | grep "lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
+  fi
+
+  COMPREPLY=( $(compgen -W "$completions" -- "$word") )
```

Comment:
I think adding actions and commands would be a great 2nd iteration on this feature :+1: